ios/Codeforces Watcher/Application/components/buttons/fabButton/FabButtonView.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+struct FabButtonView: View {
+    
+    var name: String? = nil
```

Comment:
I'm wondering if this should be passed as constructor params and be not optional.